From 55683f54ebabd30a384f3b75bcddaa3777a8aafa Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sun, 5 Nov 2017 14:46:06 +0000
Subject: [PATCH 1/1] linux: use python3

* In Solus streamlink is built with python3 but the default is python2,
  use python3 by default
---
 src/config/streamprovider.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/config/streamprovider.json b/src/config/streamprovider.json
index b542b0fa..3338627c 100644
--- a/src/config/streamprovider.json
+++ b/src/config/streamprovider.json
@@ -27,11 +27,11 @@
 			"flavor": "default",
 			"name": "Streamlink",
 			"label": "Streamlink",
-			"python": ">=2.6 <3 || >=3.3",
+			"python": ">=2.6 <3 || >=3.6",
 			"exec": {
 				"win32": "pythonw.exe",
 				"darwin": "python",
-				"linux": "python"
+				"linux": "python3"
 			},
 			"fallback": {
 				"win32": [
-- 
2.15.0

